src/redismodule.rs

```diff
@@ -7,6 +7,7 @@ use std::string::FromUtf8Error;
 pub use crate::raw;
 pub use crate::rediserror::RedisError;
 pub use crate::redisvalue::RedisValue;
+use bitflags::_core::str::Utf8Error;
```

Comment:
1. The import is wrong, should be `use std::str::Utf8Error`.
2. Also, please put it higher up together with other `std` imports.
3. Finally, as long as you've imported it, please change the `from_ptr` return type accordingly.